src/patterns/scheduled-ecs-task.test.ts

```diff
@@ -0,0 +1,67 @@
+import "@aws-cdk/assert/jest";
+import { SynthUtils } from "@aws-cdk/assert";
+import { SecurityGroup, Vpc } from "@aws-cdk/aws-ec2";
+import { Schedule } from "@aws-cdk/aws-events";
+import { Effect, PolicyStatement } from "@aws-cdk/aws-iam";
+import { Duration } from "@aws-cdk/core";
+import { simpleGuStackForTesting } from "../utils/test";
+import { GuScheduledEcsTask } from "./scheduled-ecs-task";
+
+describe("The GuScheduledEcsTask pattern", () => {
+  it("should create the correct resources with minimal config", () => {
+    const stack = simpleGuStackForTesting();
+
+    const vpc = Vpc.fromVpcAttributes(stack, "VPC", {
+      vpcId: "test",
+      availabilityZones: [""],
+      publicSubnetIds: [""],
+      privateSubnetIds: ["abc-123"],
+    });
+    new GuScheduledEcsTask(stack, {
+      schedule: Schedule.rate(Duration.minutes(1)),
+      containerConfiguration: { id: "node:10" },
+      vpc: vpc,
+      stack: "test",
+      stage: "TEST",
+      app: "ecs-test",
+    });
+    expect(SynthUtils.toCloudFormation(stack)).toMatchSnapshot();
+    expect(stack).toHaveResource("AWS::IAM::Policy");
```

Comment:
This assertion feels a bit odd - the snapshot assertion above it should provide enough confidence that an IAM Policy has been created? Unless we want to assert on specific properties of the Policy?